api/buckets/service.go

```diff
@@ -59,6 +62,25 @@ var (
 
 	// errInvalidNodeType indicates a node with type other than raw of proto was encountered.
 	errInvalidNodeType = errors.New("invalid node type")
+
+	// baseArchiveStorageConfig is used as
+	baseArchiveStorageConfig = ffs.StorageConfig{
+		Hot: ffs.HotConfig{
+			Enabled: false,
+		},
+		Cold: ffs.ColdConfig{
+			Enabled: true,
+		},
+	}
+
+	// ToDo: Export the default storage config from powergate so we can create this from it.
+	defaultDefaultArchiveConfig = mdb.ArchiveConfig{
+		RepFactor:       5,
+		TrustedMiners:   []string{"t016303", "t016304", "t016305", "t016306", "t016309"},
+		DealMinDuration: powUtil.MinDealDuration * 2,
+		FastRetrieval:   true,
+		DealStartOffset: 72 * 60 * 60 / powUtil.EpochDurationSeconds, // 72hs
+	}
```

Comment:
If the user hasn't set a default `ArchiveConfig` for their Bucket, this one is used. I'd like to export the default `StorageConfig` from Powergate so we can create this dynamically based on whatever is set in Powergate.